docs/pages/setup/admin/graceful-restarts.mdx

```diff
@@ -1,53 +1,50 @@
 ---
-title: Graceful Restarts
-description: Graceful restarts of Teleport.
+title: Upgrading the Teleport Binary
+description: How to upgrade a teleport binary without sacrificing availability.
 ---
 
-<Notice scope={["cloud"]} type="tip">
+In this guide, we will show you how to upgrade the `teleport` binary on a
+virtualized or bare-metal Linux host without sacrificing availability.
```

Comment:
Would just "Linux host" be fine here ? It feels like virtualised or bare-metal basically encompasses all Linux environments. If the idea is to try and distinguish this guide as being inappropriate for upgrading Teleport when it's running as a container, there might be some better language for us to use.